src/components/dock/dock-button/dock-button.scss

```diff
@@ -73,3 +73,37 @@ limel-badge {
     top: -0.125rem;
     right: -0.125rem;
 }
+
+// 👇 Below is a hidden and private hack
```

Comment:
I guess it will no longer be a 'private hack' so perhaps this should be updated accordingly?

Reply:
Ah I see. 😅